sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/Cast.java

```diff
@@ -42,4 +42,9 @@ public Cast(Expression expression, DataType dataType) {
 
   @Override
   public Expression[] children() { return new Expression[]{ expression() }; }
+
+  @Override
+  public String toString() {
```

Comment:
is this the only missing one? maybe we should add a default implementation in the base class `Expression` using `ToStringSQLBuilder`?

Reply:
No, some V2 expressions missing `toString` too. e.g. `Extract`.
Add a default implementation in the base class Expression using `ToStringSQLBuilder` is a good idea.

Reply:
I will fix it.